examples/basic_operations/get_responsive_search_ads.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example gets non-removed responsive search ads in a specified ad group.
+
+To add responsive search ads, run basic_operations/add_responsive_search_ad.py.
+To get ad groups, run basic_operations/get_ad_groups.py.
+"""
+
+import argparse
+import sys
+
+from google.ads.google_ads.client import GoogleAdsClient
+from google.ads.google_ads.errors import GoogleAdsException
+
+
+_DEFAULT_PAGE_SIZE = 1000
+
+
+def main(client, customer_id, page_size, ad_group_id=None):
+    ga_service = client.get_service('GoogleAdsService', version='v2')
+
+    query = '''
```

Comment:
**Nitpick**
I don't feel strongly about it, but I generally recommend against the usage of triple-quote strings for most cases because it includes all the whitespace used for proper indentation in Python needlessly as part of the string. An equivalent would be something like:

```
    query = ('SELECT ad_group.id, ad_group_ad.ad.id, '
             'ad_group_ad.ad.responsive_search_ad.headlines, '
             'ad_group_ad.ad.responsive_search_ad.descriptions, '
             'ad_group_ad.status FROM ad_group_ad '
             'WHERE ad_group_ad.ad.type = RESPONSIVE_SEARCH_AD '
             'AND ad_group_ad.status != "REMOVED"')
```

Reply:
It was like that previously but I updated per the prior discussion. I'm going to leave it as is for now.

Reply:
Random user of this python lib here. 
After months of hating writing queries in the original example way, and then having triple quotes + f-strings dawn on me.
From everyday dev writing happiness, triple quotes FTW! Thank you